python/doc/theory/meta_modeling/regression_analysis.rst

```diff
@@ -0,0 +1,197 @@
+Experimental design
+~~~~~~~~~~~~~~~~~~~
+
+A set of observations of the input parameters is required:
+
+.. math::
+
+    \cX = \left\{ \vect{x}^{(1)}, \dots, \vect{x}^{(N)} \right\},
+
+as well as the corresponding model evaluations:
+
+.. math::
+
+    \cY = \left\{ y^{(1)},\dots,y^{(N)} \right\}.
+
+Let :math:`\vect{y} = (y^{(1)},\dots,y^{(N)})^T \in \Rset^{N}` be the
+vector of output observations.
+
+Statistics
+~~~~~~~~~~
+
+Let :math:`\bar{y}` be the sample mean:
+
+.. math::
+
+    \bar{y} = \frac{1}{n} \sum_{j = 1}^n y_j.
+
+The total sum of squares ([baron2014]_ page 398) is:
+
+.. math::
+
+    SS_{TOT}
+    = \sum_{j = 1}^n \left(y_j - \bar{y}_j\right)^2
+    = \left(\vect{y} - \bar{\vect{y}}\right)^T \left(\vect{y} - \bar{\vect{y}}\right).
+
+where :math:`\hat{\vect{y}} = (\bar{y}, ..., \bar{y})^T \in \Rset^n`.
+
+The regression sum of squares is:
+
+.. math::
+
+    SS_{REG}
+    = \sum_{j = 1}^n \left(\hat{y}_j - \bar{y}_j\right)^2
+    = \left(\hat{\vect{y}} - \bar{\vect{y}}\right)^T \left(\hat{\vect{y}} - \bar{\vect{y}}\right).
+
+The error sum of squares is:
+
+.. math::
+
+    SS_{ERR}
+    = \sum_{j = 1}^n \left(y_j - \hat{y}_j\right)^2
+    = \left(\vect{y} - \hat{\vect{y}}\right)^T \left(\vect{y} - \hat{\vect{y}}\right).
+
+Coefficient of determination
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The coefficient of determination is ([baron2014]_ page 399):
+
+.. math::
+
+    R^2 = \frac{SS_{REG}}{SS_{TOT}}.
+
+The coefficient of determination measures the part
+of the variance explained by the linear regression model.
+
+Variance
+~~~~~~~~
+
+The unbiased estimator of the variance is ([baron2014]_ page 400,
+[bingham2010]_ page 67):
+
+.. math::
+
+    \hat{\sigma}^2 = \frac{SS_{ERR}}{n - p}.
+
+ANOVA F-test
+~~~~~~~~~~~~
+
+The F-statistic is based on the hypothesis that all coefficients
+are simultaneously zero ([baron2014]_ page 400).
+The ANOVA F-test is based on the hypothesis:
+
+.. math::
+
+    H_0 : a_1 = . . . = a_p = 0
+    \qquad \textrm{vs} \qquad
+    H_A : \textrm{at least one } a_k \neq 0.
+
+Let :math:`n \in \Nset` be the sample size.
+Let :math:`\vect{y} \in \Rset^n` be the vector of
+observations and :math:`\vect{\hat{y}} \in \Rset^n`
+be the vector of predictions.
+
+The F-statistic is ([bingham2010]_ Kolodziejcyzk’s theorem 6.5 page 154,
+[baron2014]_ page 400):
+
+.. math::
+
+    F = \frac{SS_{reg} / p}{SS_{ERR} / (n - p)}.
+
+The p-value is computed from the Fisher-Snedecor
+distribution :math:`F_{p, n - p - 1}` ([baron2014]_ page 400, [faraway2014]_ page 35).
+
+T-test for individual coefficients
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let :math:`\hat{\sigma}^2` be the estimator of the variance ([baron2014]_ page 400):
+
+.. math::
+
+    \hat{\sigma}^2 = \frac{SS_{ERR}}{n - p}.
+
+Let :math:`\hat{\sigma}^2(a_k)` be the diagonal of the
+covariance matrix :math:`\sigma^2 (\Psi^T \Psi)^{-1}`:
+
+.. math::
+
+    \hat{\sigma}^2(a_k) = \hat{\sigma}^2 \left(\Psi^T \Psi\right)^{-1}_{kk}
+
+for :math:`k = 1, ..., p`.
```

Comment:
I believe there are several problems here:

1. $\Psi$ is not defined, and should probably use the \mat macro.
2. If the goal is to define $a_k$, then you should probably get rid of the $\hat{\sigma}^2$ factor which pollutes the definition.
3. $a_k$ is the $k$-th diagonal coefficient of the matrix, not the whole diagonal.

Reply:
Very true. And use the `\Tr` macro.

Reply:
Fixed at be1335ae7